lib/resources/r4/document_reference.yaml

```
@@ -2,12 +2,82 @@
 name: DocumentReference
 field_name_base_url: https://www.hl7.org/fhir/documentreference-definitions.html#DocumentReference
 fields:
+- name: resourceType
+  required: 'Yes'
+  type: string
+  description: The type of the FHIR resource.
+  example: |
+    {
+      "resourceType": "DocumentReference"
+    }
+  note: resourceType must be DocumentReference.
+  url: http://hl7.org/fhir/r4/documentreference.html
+
+- name: status
+  required: 'Yes'
+  type: code
+  action:
+    - create
+  description: The status of this document reference.
+  note: Only `current` is supported
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.status
+  example: |
+    {
+      "status": "current"
+    }
+  binding:
+    description: The status of the document reference.
+    terminology:
+      - display: DocumentReference status
+        system: http://hl7.org/fhir/document-reference-status
+        info_link: https://www.hl7.org/fhir/valueset-document-reference-status.html
+
+- name: docStatus
+  required: 'Yes'
+  type: code
+  action:
+    - create
+  description: The status of the underlying document. Must be a supported Composition Status
```

```suggestion
  description: The status of the underlying document.
```
nit: I think we make it clear on what's supported in the note.

---

You can find the changes here: https://github.com/cerner/fhir.cerner.com/pull/446/commits/134ae0b5f90e1a42b6d4c1e53e7cd966432c4fee